pkgs/applications/networking/mumble/default.nix

```diff
@@ -121,7 +121,6 @@ let
         "-D bundled-gsl=OFF"
         # "-D bundled-renamenoise=OFF" # not packaged in nixpkgs yet
         "-D update=OFF"
-        "-D overlay-xcompile=OFF"
```

Comment:
But these options are for the client? So why removing it?

Reply:
This commit should be squashed with the follow-up, [mumble: add overlay=False build option as we have our own target for the overlay](https://github.com/NixOS/nixpkgs/pull/384691/commits/90f24d2657f3cba17004a7fce3348c239cb05052), since this commit alone breaks building mumble.

However, by disabling Overlay support its related menu disappears from the Mumble settings window, and I assume that provides easy use of it. So this might be a breaking change. Unless we figure out how we can bring it back with the mumble_overlay package, I'm rather negative about disabling it.

What's your reason for disabling overlay? Is it needed for MacOS that it compiles or just for cleaning up the build options? In case of MacOS, we could disable it specifically for that platform.